addons/mail/static/src/components/chatter/chatter_tests.js

```diff
@@ -452,6 +461,160 @@ QUnit.test('do not post message with "Enter" keyboard shortcut', async function
     );
 });
 
+QUnit.test("Display suggested recipients list", async function (assert) {
+    assert.expect(3);
+
+    this.data['res.partner'].records.push({ id: 100, email: 'john@jane.be', display_name: 'John Jane' });
+    this.data['res.fake'].records.push({
+        id: 10,
+        email_cc: "john@test.be",
+        partner_ids: [100],
+    });
+    await this.start ();
+    const chatter = this.env.models['mail.chatter'].create({
+        threadId: 10,
+        threadModel: 'res.fake',
+    });
+    await this.createChatterComponent({ chatter });
+
+    await afterNextRender(() =>
+        document.querySelector(`.o_ChatterTopbar_buttonSendMessage`).click()
+    );
+
+    assert.containsOnce(
+        document.body,
+        '.o_ComposerSuggestedRecipientList',
+        "Should display a list of suggested recipients after opening the composer from 'Send message' button"
+    );
+    assert.containsN(
+        document.body,
+        '.o_ComposerSuggestedRecipient',
+        3,
+        "There should be 3 suggested recipients"
+    );
```

Comment:
This looks out of place of this test. Doesn't explain why, so very poorly written test.

Move this in another test, dedicated to limiting the amount of recipients to display to 3 by default when there are more than 3 suggested recipients.